Sources/SFSafeSymbols/Symbols/SFSymbol+2020.1.swift

```diff
@@ -0,0 +1,153 @@
+// Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
+
+// 2020.1 Symbols
+@available(iOS 14.2, macOS 11.0, tvOS 14.2, watchOS 7.1, *)
+public extension SFSymbol {
+    /// 􀵀
+    static let aqiHigh = SFSymbol(rawValue: "aqi.high")
+
+    /// 􀴾
+    static let aqiLow = SFSymbol(rawValue: "aqi.low")
+
+    /// 􀴿
+    static let aqiMedium = SFSymbol(rawValue: "aqi.medium")
+
+    /// 􀮰
+    static let brazilianrealsignCircle = SFSymbol(rawValue: "brazilianrealsign.circle")
+
+    /// 􀮱
+    static let brazilianrealsignCircleFill = SFSymbol(rawValue: "brazilianrealsign.circle.fill")
+
+    /// 􀮲
+    static let brazilianrealsignSquare = SFSymbol(rawValue: "brazilianrealsign.square")
+
+    /// 􀮳
+    static let brazilianrealsignSquareFill = SFSymbol(rawValue: "brazilianrealsign.square.fill")
+
+    /// 􀒭
+    static let cartCircle = SFSymbol(rawValue: "cart.circle")
+
+    /// 􀒮
+    static let cartCircleFill = SFSymbol(rawValue: "cart.circle.fill")
+
+    /// 􀅏
+    /// From iOS 14.2, macOS 11.0, tvOS 14.2 and watchOS 7.1 on, the following localizations are available: Arabic, Chinese, Hebrew, Hindi, Japanese, Korean, Thai, Traditional Chinese
+    static let character = SFSymbol(rawValue: "character")
```

Comment:
System requirements is too high. Users will be forced to use deprecated name. See https://github.com/piknotech/SFSafeSymbols/pull/71#issuecomment-803301090

Reply:
what are the disadvantages of using the deprecated name? The warning will appear only when the deployment target is >= 14.2
Even the redesigned symbols in 14.5 will be used when using the old names. [source](https://hacknicity.medium.com/sf-symbol-changes-in-ios-14-5-3fb3ed068241)
Not sure about localization though.

Reply:
Just tested on a 14.4 simulator. Localization works with old names.

Reply:
@StevenMagdy You're right in a sense, but I think the better way would be to also allow using the `character` name even when targeting something lower than 14.2. A user might browse the SF Symbols app, see the `character` symbol and want to use it in their app. Now, instead of using `SFSymbol.a` (which would work), the more natural kind of usage would be `SFSymbol.character` because that's how it is called nowadays and the user should not worry about whether using the latest name works when targeting older operating systems. That's also how it works with the current implementation.

I suggest the following approach:

In `SFSymbol+2019.swift`:

```swift
// ...
static let a: SFSymbol {
    if #available(iOS 14.2, macOS 11.0, tvOS 14.2, watchOS 7.1, *) {
        return SFSymbol(systemName: "character") // "a" would also work, but this is more clean
    } else {
        return SFSymbol(systemName: "a")
    }
}

static let character: SFSymbol {
    return SFSymbol.a
}

// ...
```

In `SFSymbol+2020.1.swift`: No changes – only newly added symbols reside in this type of file. All renames are managed in the file of the year when the symbol was first added.

What do you think?

Reply:
@fredpi You are 100% right about the user browsing the app and not being able to use `character`. Sorry I missed that.

I like your approach a lot. I suggest keeping `a` deprecated to encourage moving to the new name, and keeping `a` a stored property:

```swift
@available(iOS, introduced: 13.0, deprecated: 14.2, renamed: "character")
static let a: SFSymbol = {
    if #available(iOS 14.2, macOS 11.0, tvOS 14.2, watchOS 7.1, *) {
        return SFSymbol(systemName: "character") // "a" would also work, but this is more clean
    } else {
        return SFSymbol(systemName: "a")
    }
}()

static var character: SFSymbol {
    return SFSymbol.a
}
```
What do you think?

Reply:
@StevenMagdy I forgot to write down the deprecation line, but meant it 😁

Also we must of course use `static var` because `static let: ... { ... }` isn't correct Swift syntax.